### PR TITLE
Remove AppVeyor and Travis-CI

### PR DESCRIPTION
**Pull request by ocefpaf**
We need to add some extra tests on GHA before merging this one.

### CODE REVIEW

File: .github/workflows/test_code.yml [hunk 1]
```diff
@@ -1,44 +1,36 @@
...
+  push:
+    branches: [master]
 
 jobs:
   run:
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
-        os: [ubuntu-latest, windows-latest]
-        python-version: [3.6, 3.9]
```
Comment (Conengmo) [inline .github/workflows/test_code.yml:17]: we should still support version 3.6, so I'd like to still test for it.

Comment (ocefpaf) [reply .github/workflows/test_code.yml:17]: I'm just testing to see which ones fail the selenium tests. We will need to skip those tests on those py versions.

**Comment by ocefpaf** [comment]
> We need to add some extra tests on GHA before merging this one.

I'll have to separate the selenium tests from the rest of the tests b/c they only run on 3.9 for some reason.

**Comment by ocefpaf** [comment]
@Conengmo this is ready for review now. The only missing pieces are the coding standards and documentation. I'm working on docs for both folium and branca and coding standards won't be needed if we adopt pre-commit hooks.

**Review by Conengmo — APPROVED** [review]

**Pull request merged**